.github/workflows/ubuntu.yml

```diff
@@ -210,7 +210,7 @@ jobs:
 
         cmake -S . -B build            \
           -DCMAKE_VERBOSE_MAKEFILE=ON  \
-          -DWarpX_APP=OFF              \
+          #-DWarpX_APP=OFF              \
```

Comment:
I had to comment this out because we don't have a guard that checks whether `WarpX_APP` is defined when we add the no-Python tests. We might want to add a check for that in case someone compiles with `-DWarpX_APP=OFF` locally. If that is the case, the configuration currently fails because it tries to find app executables that do not exist.